expressEngine/model/lib/index.js

```diff
@@ -3,13 +3,15 @@ class ModelLib {
     this.roleEnum = ['system-admin', 'supplier'];
   }
 
-  validateEmail = email => {
-    var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
-    return re.test(String(email).toLowerCase());
+  orderStatus =() => ['confirmed', 'shipped', 'In-Delivery', 'delivered', 'closed'];
```

Comment:
add cancelled field instead of closed

Reply:
'closed' is mention in the issue. please have a look.

Reply:
@aman-successive I think Cancel field is for order items, order is always closed.